tests/rbd_mirror/test_rbd_mirror_snapshot_metrics.py

```diff
@@ -0,0 +1,158 @@
+"""Test case covered -
+    CEPH-83575565 - Performance counter metrics for snapshot based
+    mirroring.
+
+    Pre-requisites :
+    1. Two Clusters must be up and running to create pool
+    2. We need atleast one client node with ceph-common package,
+       conf and keyring files on each node.
+
+    Test Case Flow:
+        1. Create a pool on both clusters.
+        2. Create an Image on primary mirror cluster in same pool.
+        3. Configure mirroring (peer bootstrap) between two clusters.
+        4. Enable image mode snapshot based mirroring on the pool respectively.
+        5. Start running IOs on the primary image.
+        6. verify snapshot based performnace counter metrics.
+"""
+import os
+import re
+
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def validate_snap_metrics(mirror_node):
+    """Method to validate snapshot based mirroring metrics
+
+    Args:
+        mirror_node: rbd-mirror daemon node
+    """
+    # Change directory to the base path
+    os.chdir("/var/run/ceph")
```

Comment:
Where are you running this command and the following os related commands? I mean on which node?

Reply:
modified.